### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by lawrencethomp**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

Greenstand is a growing repository with a great amount of people working it that's fighting poverty and climate change and poverty, and thus should be added to this list. The repositories I've suggested are for the overview of the repo and the Web Map, as well as the Android app.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by lawrencethomp** [comment]
I'm not sure why this failed, the error message seemed unrelated   
1) Failure:
collections::made-in-africa collection#test_0007_fails if a user, organization, or repository has been renamed [/home/travis/build/github/explore/test/collections_test.rb:90]:
Expected ["made-in-africa: praekelt/vumi has been renamed to praekeltfoundation/vumi", "made-in-africa: praekelt/junebug has been renamed to praekeltfoundation/junebug"] to be empty.
6429 runs, 151246 assertions, 1 failures, 0 errors, 0 skips
rake aborted!
Command failed with status (1)
/home/travis/build/github/explore/vendor/bundle/ruby/2.5.0/gems/rake-12.3.2/exe/rake:27:in `<top (required)>'
/home/travis/.rvm/rubies/ruby-2.5.1/bin/bundle:30:in `block in <main>'
/home/travis/.rvm/rubies/ruby-2.5.1/bin/bundle:22:in `<main>'
/home/travis/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
/home/travis/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => test
(See full trace by running task with --trace)
Inspecting 8 files

File: collections/social-impact/index.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ items:
  - IEEEKeralaSection/rescuekerala
  - Data4Democracy/ethics-resources
  - civicdata/civicdata.github.io
+ - Greenstand/Development-Overview
+ - Greenstand/treetracker-web-map
```
Comment (kenyonj) [inline collections/social-impact/index.md]:
```suggestion
```

File: collections/social-impact/index.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ items:
...
+ - Greenstand/treetracker-android
```
Comment (kenyonj) [inline collections/social-impact/index.md]:
```suggestion
```

**Comment by kenyonj** [comment]
Looks like https://github.com/github/explore/pull/496 is already following my suggestion, so I'll get that merged in and leave this open to continue the discussion.

**Comment by kenyonj** [comment]
I've merged in https://github.com/github/explore/pull/496. If you agree with my comments you can go ahead and close this pull request. Thanks again for your contributions.

**Comment by lawrencethomp** [comment]
That sounds like a fair assessment. I agree.

**Pull request closed without merging**